### PR TITLE
#1262-fix

### PR DESCRIPTION
**Pull request by robbix1206**
Add of the grass path block and correct the farmland to a 15/16 block.
I think its correct but well.. it's my first commit so please if someone can verify what I done.

### CODE REVIEW

File: overviewer_core/textures.py [hunk 1]
```diff
@@ -2167,13 +2167,41 @@ def crops(self, blockid, data):
...
-@material(blockid=60, data=range(9), solid=True)
-def farmland(self, blockid, data):
-    top = self.load_image_texture("assets/minecraft/textures/blocks/farmland_wet.png")
-    if data == 0:
-        top = self.load_image_texture("assets/minecraft/textures/blocks/farmland_dry.png")
-    return self.build_block(top, self.load_image_texture("assets/minecraft/textures/blocks/dirt.png"))
+# farmland and grass path (15/16 blocks)
+@material(blockid=[60,208], data=range(9), solid=True)
+def farmland_and_grass_path(self, blockid, data):
+    if blockid=60:
```
Comment (agrif) [inline overviewer_core/textures.py]: You probably mean `if blockid == 60` here. Other than that, I can't see anything obviously out of place.

Comment (robbix1206) [inline overviewer_core/textures.py]: Thanks you haven't seen it ><
I took the daylight_sensor as model and copy it and modify it.

**Comment by robbix1206** [comment]
Doesn't interference why texture findre crash ><

**Comment by robbix1206** [comment]
Was thinking of Travis configure for 1.8 at first but after looking at the log overviewer can't access to the grass picture ><

**Comment by agrif** [comment]
Since this is a 1.9 block, we'll either merge this in a snapshot branch to eventually be merged when 1.9 is released, or just wait to merge this when 1.9 is released.

Either way, travis should work itself out once it actually uses the 1.9 jars.

**Comment by NoxNebula** [comment]
The path Blocks are not 15/16, it seems they are 1/2 block, as high as a slab.

**Comment by eminence** [comment]
I'm looking at http://minecraft.gamepedia.com/Grass_Path which says they are 15/16th of a block.  Has that changed?

**Comment by NoxNebula** [comment]
I mean, they are 1/2 at the overviewer, but in minecraft 15/16

**Comment by eminence** [comment]
Closed via 5f9276a24a8d64efe4b91cf29c565b4541a0cdf0

**Comment by NoxNebula** [comment]
They are now ok :+1:

**Pull request closed without merging**